settings/default/main.lua

```diff
@@ -25,6 +25,8 @@ xi.settings.main =
     RESTRICT_CONTENT = 0,
 
     -- Enable Expansion (1 = Enabled, 0 = Disabled)
+    ENABLE_VANILLA   = 1;
```

Comment:
I don't think that exposing this is a good idea, why would anyone want to disable the base zones.

Also, seeing as the English client launched with ROZ, it's not super useful to have this additional tag

Reply:
It's to stop the IsContentEnabled function from making an error. It has a line where it's going through the settings file and checking it for the content tag pass to it. In this case the content tag is vanilla for many of the zones since they are vanilla zones and then the function produces an error because it can't find vanilla in main.lua

I could just have it returned true if it receives a value of vanilla or Roz